### PR TITLE
Autocomplete dartpad examples fix

### PR DESCRIPTION
**Pull request by justinmc**
The dartpad examples for Autocomplete and RawAutocomplete in the docs weren't rendering (e.g. https://master-api.flutter.dev/flutter/widgets/RawAutocomplete-class.html).

They seemed to be poorly formatted by looking at other freeform examples that do work.  This PR will hopefully get them to work by copying a working format.

No tests, just a docs fix.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/autocomplete.dart [hunk 1]
```diff
@@ -279,146 +327,139 @@ typedef AutocompleteOptionToString<T extends Object> = String Function(T option)
...
+///               setState(() {
+///                 _autocompleteSelection = selection;
+///               });
+///             },
+///             fieldViewBuilder: (BuildContext context, TextEditingController textEditingController, FocusNode focusNode, VoidCallback onFieldSubmitted) {
+///               return TextFormField(
+///                 controller: textEditingController,
 ///                 decoration: InputDecoration(
-///                   hintText: 'This is a regular TextFormField',
+///                   hintText: 'This is an RawAutocomplete!',
```
Comment (shihaohong) [inline packages/flutter/lib/src/widgets/autocomplete.dart:387]: nit
```suggestion
///                   hintText: 'This is a RawAutocomplete!',
```

**Review by shihaohong — APPROVED** [review]
LGTM, just a tiny nit

**Comment by shihaohong** [comment]
The analyzer is complaining about not using `const`s in the sample docs. I wonder if there's an easier way to automatically make some of these corrections. I find that it's very difficult to edit code in the code samples

**Comment by justinmc** [comment]
Google testing seems to be broken, and this is just a docs change and should have no effect, so I'll go ahead and merge this.

**Pull request merged**